### DISCUSSION TITLE
What is DEX crossover operator?

### DISCUSSION THREAD

**Discussion by nulogar** [anyoptimization/pymoo#422] [Q&A] [answered]
Hello to everyone,

Could someone tell me which crossover operator is exactly DEX of pymoo.operators.crossover.dex? I've looked for it in the online documentation, but didn't find anything.

Thanks in advance.

**Comment by blankjul** [comment, marked as answer]
It is the Differential Evolution Crossover used in DE: https://pymoo.org/algorithms/soo/de.html

**Comment by nulogar** [comment]
Thank you.

Since it is an isolated crossover operator, can it be combined for use with any algorithm and mutation/selection operator?  Could it lead to unwanted interactions?

**Reply by blankjul** [reply]
In theory yes. But it has been developed for DE also does not need only 2 parents like other operators.
